sdk/core/perf-tests/core-rest-pipeline/tsconfig.json

```diff
@@ -1,13 +1,21 @@
 {
   "extends": "../../../../tsconfig",
   "compilerOptions": {
-    "outDir": "./dist-esm",
     "module": "NodeNext",
     "moduleResolution": "NodeNext",
     "target": "ESNext",
-    "lib": ["ESNext", "DOM"]
+    "paths": {
+      "@azure-tests/perf-core-rest-pipeline": ["./src/index"]
```

Comment:
Do we need this? I think the alias is mainly for samples

Reply:
It can't hurt.